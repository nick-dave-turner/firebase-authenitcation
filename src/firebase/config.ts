/* istanbul ignore file */
interface FirebaseConfig {
  apiKey: string | undefined;
  appId: string | undefined;
  authDomain: string | undefined;
  databaseURL: string | undefined;
  measurementId: string | undefined;
  messagingSenderId: string | undefined;
  projectId: string | undefined;
  storageBucket: string | undefined;
}

export const firebaseConfig: FirebaseConfig = {
  apiKey: process.env.REACT_APP_API_KEY,
  appId: process.env.REACT_APP_APP_ID,
  authDomain: process.env.REACT_APP_AUTH_DOMAIN,
  databaseURL: process.env.REACT_APP_DATABASE_URL,
  measurementId: process.env.REACT_APP_MEASUREMENT_ID,
  messagingSenderId: process.env.REACT_APP_MESSAGING_SENDER_ID,
  projectId: process.env.REACT_APP_PROJECT_ID,
  storageBucket: process.env.REACT_APP_PROJECT_ID,
};
