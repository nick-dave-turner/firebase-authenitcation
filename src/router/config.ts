/* istanbul ignore file */
/* eslint-disable @typescript-eslint/no-unused-vars */
import { lazy } from "react";
import { RoutingProps } from "../components";

export const routerConfig: RoutingProps[] = [
  {
    component: lazy(() => import("../pages/home")),
    path: "/home",
    title: "Home",
  },
  {
    component: lazy(() => import("../pages/private")),
    path: "/private",
    title: "Hidden",
  },
];
