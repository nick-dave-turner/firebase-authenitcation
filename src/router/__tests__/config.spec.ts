import { routerConfig } from "../config";

describe("routerConfig", () => {
  it("should match snapshot", () => {
    expect(routerConfig).toMatchSnapshot();
  });
});
