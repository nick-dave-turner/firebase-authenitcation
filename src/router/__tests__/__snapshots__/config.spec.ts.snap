// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`routerConfig should match snapshot 1`] = `
Array [
  Object {
    "component": Object {
      "$$typeof": Symbol(react.lazy),
      "_init": [Function],
      "_payload": Object {
        "_result": [Function],
        "_status": -1,
      },
    },
    "path": "/home",
    "title": "Home",
  },
  Object {
    "component": Object {
      "$$typeof": Symbol(react.lazy),
      "_init": [Function],
      "_payload": Object {
        "_result": [Function],
        "_status": -1,
      },
    },
    "path": "/private",
    "title": "Hidden",
  },
]
`;
